hardhat-deploy/contracts/Cars.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
Rather than use these contracts, I think it would be better to keep the example more concise, since the prupose is only to demonstrate a chained deployment.

So here is what I'd suggest:
- Create a `Foo` contract that does nothing except for accept `Bar` (address) as a constructor argument
  - require that it is a non-zero address, and store it in a state variable.
- The `Bar` contract can be empty, does not need to do anything.